go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/rickb777/date v1.12.5-0.20200422084442-6300e543c4d9
 	github.com/sergi/go-diff v1.2.0
 	github.com/tombuildsstuff/giovanni v0.20.0
-	github.com/tombuildsstuff/kermit v0.20221125.1100802
+	github.com/tombuildsstuff/kermit v0.20221207.1110610
```

Comment:
SDK updates are handled separately to API version updates, can we revert this in favour of #19698